configure.ac

```diff
@@ -332,15 +332,15 @@ AC_DEFINE_UNQUOTED(SBD_WATCHDOG_TIMEOUT_DEFAULT, $SBD_WATCHDOG_TIMEOUT_DEFAULT,
     Timeout in seconds SBD will configure the watchdog per default)
 
 if test x"${SBD_SYNC_RESOURCE_STARTUP_DEFAULT}" = x""; then
-    SBD_SYNC_RESOURCE_STARTUP_DEFAULT=no
+    SBD_SYNC_RESOURCE_STARTUP_DEFAULT=yes
 fi
 AC_SUBST(SBD_SYNC_RESOURCE_STARTUP_DEFAULT)
 dnl rather pass to C as a string and interpret there for consistent interpretation
 AC_DEFINE_UNQUOTED(SBD_SYNC_RESOURCE_STARTUP_DEFAULT, "${SBD_SYNC_RESOURCE_STARTUP_DEFAULT}",
     Default for SBD_SYNC_RESOURCE_STARTUP if not given in SBD configuration file explicitly)
 
 if test x"${SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG}" = x""; then
-    SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG=no
+    SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG=yes
```

Comment:
Not important, but SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG=${SBD_SYNC_RESOURCE_STARTUP_DEFAULT} might make sense (i.e. default to the default)

Reply:
good point 

definitely a logic that makes sense
but doesn't map very well to the with/without-logic in the spec-file

maybe having just the default exposed for with/without would make sense +
in addition a global for the sysconfig-value that is empty/auto/yes/no to
still allow explicit setting